pyseir/models/suppression_policies.py

```diff
@@ -22,6 +22,31 @@
 }
 
 
+def get_future_supression_from_r0(R0, scenario):
+    """
+    Returns the future suppression level for a given R0 and a "future scenario".
+
+    Parameters
+    ----------
+    R0:float
+        Reproduction number
+    scenario: str
+        'no_intervention', 'flatten_the_curve', 'social_distancing'.
+    Returns
+    -------
+    epsilon: float
+        Suppression percentage.
+    """
+    if scenario == 'no_intervention':
+        return 1
+    elif scenario == 'flatten_the_curve':
+        return 0.97 / R0
+    elif scenario == 'social_distancing':
+        return 1.7 / R0
```

Comment:
It is unclear to me how value for social distancing was chosen ?

Reply:
Same :)... They were largely from some early studies max found in Wuhan.  We will be updating once we have some better reopening scenarios.  There are refs in the modeling doc.